### ISSUE TITLE
[Idea] Better terminology for certain words (Better translations)

### ISSUE THREAD

**Issue by Andre601** [go-gitea/gitea#23280]
### Feature Description

This has originally been suggested on [forgejo/forgejo @Codeberg](https://codeberg.org/forgejo/forgejo/issues/330) but I was told it would be a good idea to propose this to the upstream repository forgejo is based of, so here we go.

I love the fact that Gitea allows translations for the UI, as it makes understanding certain things a lot easier.
However, certain words cannot be translated that easily, either because there isn't a translation available, or the translation wouldn't be as suitable.

Take for example "Issues". In german, my native language, would it translate to "Probleme", which has more of a negative tone to it, which is why the word is used untranslated in german. And while issues are mainly for... well... issues, have issue tracker gone way beyond their initial purpose and are now also used for feature requests (Such as this one), but also questions or even general feedback.

The term "issues" is therefore not really that up-to-date anymore and a suitable replacement should be considered.
My personal suggestion would be "Tickets" as a ticket is more neutral than an issue.
Also, given my usage of Forgejo, which to my knowledge is mostly the same as gitea in terms of design, does the Activity history you can see already use "Ticket" as a word in translations, to refer to issues, which means that a change could make it more consistent.

Of course, there are a few other terms that aren't translated and used as-is for various reasons, so it should be considered to look into this and see what words and/or phrases can be improved to make translations that little bit better for everyone.

**Comment by jolheiser** [comment]
Our translations are currently done via [crowdin](https://crowdin.com/project/gitea), if you have specific ideas we could probably add them there and have them reviewed.

Changing them in code or across languages might be a bit larger of an undertaking, however.

**Comment by jolheiser** [comment]
Interestingly, crowdin seems to suggest `Probleme` for a no-context `Issues`, however for something like `Create Issue` it suggests `Ticket erstellen`.

(I know precisely zero German, I just found it interesting given your feedback above `Probleme` vs `Tickets`)

**Comment by Andre601** [comment]
> Interestingly, crowdin seems to suggest `Probleme` for a no-context `Issues`, however for something like `Create Issue` it suggests `Ticket erstellen`.
> 
> (I know precisely zero German, I just found it interesting given your feedback above `Probleme` vs `Tickets`)

Yep. I found it interesting to see that "Issues" was used for the tab, while the overview (Page showing past activities) is using "Ticket".
It confused me and made me consider this proposal to have the word changed to Ticket in general (And by that I don't just mean translations, but source itself. That kinda was missed out in my OP) for the previously mentioned points of issues no longer being "just for issues".

Also, other software for bug-report and feature request management - like Jira - also use Tickets as their main words (Tho, Idk if that is also the case in original and not just translations), which for me makes sense.

**Comment by lunny** [comment]
We also have a German channel in discord, maybe you can find more German native speaker to discuss how to translate it well.

**Comment by krzysztofjeziorny** [comment]
I just noticed the German translation of _Issues_ in 1.27.0. As said, "Probleme" has here a quite negative tone, and can cause confusion to the users not familiar with Gitea or ticketing systems in general. These would be clients mostly, which I encourage to use Gitea for easier communication. For them creating a new "Problem" is a weird thing, as describing ideas or solutions is not a problem.

If I may suggest here, as I don't have Discord, translating "Issues" into "Tickets" or just "Aufgaben" (Tasks), that would be beneficial to better understanding what Issues are good for.

**Comment by TheFox0x7** [comment]
As an alternative name idea, Gitlab changed their "issues" to "work items".
